CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+##[Unreleased] 2016-05-25
+### Added
+- #25 class RequestContext (alternate of `setData` function in `Form` class)
+- #25 function `handleRequest($request)` to `Form` class
+- #25 function `handleRequest($request)` to `BaseElement` class
+- #25 function `clearValue()` to `ElementInterface`
+- function `isChecked($value)` to `CheckBoxList` class
```

Comment:
CheckBoxList::getValue return all selected values. But what if you want to know some value is checked?